bin/ui5linter.js

```diff
@@ -42,10 +42,17 @@ const { minimatch } = require("minimatch");
 					return new Promise(resolve => resolve());
 				}
 
-				const bShouldXmlFormatterTagEndByNewline = process.argv.includes("--tagEndNewline");
-				const bShouldXmlFormatterTagSpaceBeforeSelfClose = process.argv.includes("--tagSpaceBeforeSelfClose");
+				const shouldXmlFormatterTagEndByNewline = process.argv.includes("--tagEndNewline");
+				const shouldXmlFormatterTagSpaceBeforeSelfClose = process.argv.includes("--tagSpaceBeforeSelfClose");
+				const indentationArg= process.argv.find(value => value.startsWith("--indentation="));
+				const indentationValue = indentationArg?.substring("--indentation=".length, indentationArg.length);
+				const indentation = indentationValue === "space" ? " ".repeat(4) : "\t";

```

Comment:
@iljapostnovs it's not for my use case, but I think others may appreciate it if they could also specify the amount of spaces they want to indent with.

Reply:
Makes sense, I will add this option :)